tests/test_examples.py

```diff
@@ -1630,3 +1632,153 @@ def circle_constraint(params):
         abs(surf_opt.Z_lmn[surf_opt.Z_basis.get_idx(M=-1, N=0)]) + offset,
         rtol=2e-2,
     )
+
+
+@pytest.mark.unit
```

Comment:
how slow is this test? it sounds like it should probably be a regression test

Reply:
< 10 min on my laptop.

Reply:
thats pretty long compared to most of the other unit tests, i'd suggest marking it regression

Reply:
Ok, done!